code/web/src/modules/user/Profile.js

```diff
@@ -9,54 +9,233 @@ import { Link } from 'react-router-dom'
 import { Grid, GridCell } from '../../ui/grid'
 import { H3, H4 } from '../../ui/typography'
 import Button from '../../ui/button'
-import { grey, grey2 } from '../../ui/common/colors'
+import ImageTile from '../../ui/image/Tile'
+import Card from '../../ui/card/Card'
+import { level3 } from '../../ui/common/shadows'
+import { white, black, grey, grey2 } from '../../ui/common/colors'
 
 // App Imports
+import { APP_URL } from '../../setup/config/env'
 import userRoutes from '../../setup/routes/user'
 import { logout } from './api/actions'
+import { getListByUser } from '../subscription/api/actions'
+import { routeImage } from '../../setup/routes'
+
+
+const userImage = `https://media.newyorker.com/photos/5e49bf473399bf0008132231/1:1/w_2539,h_2539,c_limit/Kenseth-CatProfile.jpg`
```

Comment:
These are just for making sure our static changes looked fine - we'll have to remember to remove them when we're done.